### ISSUE TITLE
Image size is too large

### ISSUE THREAD

**Issue by nelsondurairaj** [manojprabhakarn/triformv2#1]
![image](https://user-images.githubusercontent.com/78611909/213787818-700586d3-7332-4ab4-af6b-bca1875e3b63.png)
while pull the code from git it i notice repo size is too large, i think image file size is the reason, we need to change this.
@manojprabhakarn

**Comment by dinesh-hills** [comment]
@nelsondurairaj Yes the images should be optimized to stay under 1000kb without much compromising the quality.
We can use techniques like converting the images to .webp format and resizing the images into various sizes and let the browser decide which image to be used based on the need.

**Comment by manojprabhakarn** [comment]
now our fonts were taken from CDN, so that it may reduce some memory

**Comment by dinesh-hills** [comment]
@manojprabhakarn  Not as much as optimizing large images.
